scripts/list-unused-test-files.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# Abort on errors (and uninitialised variables)
+set -eu
+
+if [ -d library -a -d include -a -d tests ]; then :; else
+    echo "Must be run from mbed TLS root" >&2
+    exit 1
+fi
+
+ls tests/data_files | \
```

Comment:
:D

Even if we have rules to construct data files in `tests/data_files/Makefile` that count as referencing the data file, I'd like to consider those data files as unused if no other files reference them.

Reply:
Note this still isn't perfect, since some of the data files that get used are generated from data files that aren't used except from within the Makefile. Preferably, I'd consider as unused only those data files that aren't reachable from the actually-used Makefile targets within the target production tree.

As currently written, this script is a heuristic that will print out some files as unused, even when they are used.

Reply:
Removing this script, since it's just a heuristic and its value in the long term is limited.